mona_sdk/client.py

```diff
@@ -1054,6 +1057,41 @@ def create_openai_context_class(self, context_class, openai_api_type):
             else get_dict_result(True, app_server_response["response_data"], None)
         )
 
+    @Decorators.refresh_token_if_needed
+    def initiate_csv_upload_request(
+        self,
+        file_path,
+        context_class,
+        context_id_field=None,
+        export_timestamp_field=None,
+    ):
+        """
+        A wrapper function for initiate_csv_upload_request REST endpoint.
+        """
+
+        def custom_bad_response_handler(json_response, status_code):
+            error_message = None
+            if status_code == CLIENT_ERROR_RESPONSE_STATUS_CODE:
+                issues = json_response.get("issues", "")
+                error = json_response.get("error_message")
+                error_message = f"{error + ': ' if (error and issues) else ''}{issues}"
+
+            if status_code == SERVER_ERROR_RESPONSE_STATUS_CODE:
+                error_message = json_response.get("error_message")
+
+            return self._handle_service_error(error_message)
```

Comment:
Can error message still be none? Is it possible for none of the two options above to work?

Reply:
It's not expected, but changed the initial error_message value to "" so that the code doesn't break inside _handle_service_error.

Reply:
OK - anything you need from here or can this be resolved?